Source/Info.plist

```diff
@@ -18,6 +18,14 @@
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>ExceptionDomains</key>
+		<dict>
+			<key>www.virtusize.com</key>
```

Comment:
oh yeah, my bad. It can just work fine without adding NSAppTransportSecurity.  I will remove it.
Somehow it didn't work until I added NSAppTransportSecurity when I tested it. 
I probably didn't build the example app properly.

Reply:
Removed

Reply:
Did you retest the example app?

Reply:
Yes.
It was strange that I was able to open the SurveyMonkey link but not the Virtusize link without adding NSAppTransportSecurity after https protocols were added.

Reply:
Maybe a `DerivedData` issue, or the project was not cleaned before the build